providers/implementations/ciphers/cipher_aes_cbc_hmac_sha.c

```diff
@@ -361,6 +371,13 @@ static void *aes_cbc_hmac_sha256_newctx(void *provctx, size_t kbits,
     return ctx;
 }
 
+static void *aes_cbc_hmac_sha256_dupctx(void *provctx)
+{
+    PROV_AES_HMAC_SHA256_CTX *ctx = provctx;
+
+    return OPENSSL_memdup(ctx, sizeof(*ctx));
```

Comment:
It is strange the all the other dupctx functions check if `ctx` is NULL first - but this one doesn't. Actually it makes no difference since `OPENSSL_memdup` checks for NULL anyway so it ends up doing the right thing.

Anyway I realise this is a backport, so this is just a "FYI" comment

Reply:
yes, this is a backport.